pyproject.toml

```diff
@@ -24,13 +24,12 @@ pyobjc-core = { version = '^7.1', platform = 'darwin' }
 pyobjc-framework-cocoa = { version = '^7.1', platform = 'darwin' }
 # easyScience
 easyDiffractionLib = { git = 'https://github.com/easyScience/easyDiffractionLib.git', rev = 'sprint12' }
-easyAppLogic = { git = 'https://github.com/easyScience/easyAppLogic.git', rev = 'develop' }
-easyAppGui = { git = 'https://github.com/easyScience/easyAppGui.git', rev = 'sprint12' }
+easyApp = { git = 'https://github.com/easyScience/easyApp.git', rev = 'develop' }
 
```

Comment:
this one simplification is a great reason for merging eA Gui and eA Logic :)
So many issues coming from out of sync Gui/Logic modules...